src/components/pages/charactersPage.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import CharactersProfile from '../charactersProfile';
+
+const CharactersPage = () => (
```

Comment:
- [Optional] This component just rendered another component (CharactersProfile). This is all it does. What makes React special is the way components interact. These smaller components(CharactersPage, HomePage) at the end form an ecosystem of complex information which hinders the flexibility of your app.